transport/src/main/java/io/scalecube/transport/MessageDeserializer.java

```diff
@@ -1,9 +1,34 @@
 package io.scalecube.transport;
 
 import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufInputStream;
+import io.netty.handler.codec.DecoderException;
+import io.protostuff.ProtostuffIOUtil;
+import io.protostuff.Schema;
+import io.protostuff.runtime.RuntimeSchema;
 
-public interface MessageDeserializer {
+public final class MessageDeserializer {
 
-  Message deserialize(ByteBuf bb);
```

Comment:
Move this ``static``  block to ``MessageSchema`` , here just load ``MessageSchema.class`` and init class -- ``Class.forName(MessageSchema.class.getName())``.